common/service/config/config.go

```diff
@@ -329,9 +329,14 @@ type (
 	}
 
 	// PublicClient is config for connecting to cadence frontend
+	// Currently supporting single IP address(HostPort) and DNS and Port. Only one of them is required.
 	PublicClient struct {
 		// HostPort is the host port to connect on
 		HostPort string `yaml:"hostPort" validate:"nonzero"`
+		// DNS is the DNS name to connect on
+		DNSPort string `yaml:"DNSPort" validate:"nonzero"`
```

Comment:
Note that current the name of the config in the file is `DNSPort`.

Reply:
I just update the code, No DNSPort anymore.